plutus-ledger/src/Ledger/Constraints/OffChain.hs

```diff
@@ -412,3 +419,8 @@ processConstraint = \case
         unless (datumHash dv == dvh)
             (throwError $ DatumWrongHash dvh dv)
         unbalancedTx . tx . Tx.datumWitnesses %= set (at dvh) (Just dv)
+
+-- | The union of two values, taking the larger amount when both keys
+--   are present.
+convexUnion :: Value -> Value -> Value
```

Comment:
@michaelpj  Should this move to `Plutus.V1.Ledger.Value` ?

Reply:
Sure.